hyperapp/hello-world.js

```diff
@@ -0,0 +1,6 @@
+import {h, app} from 'https://unpkg.com/hyperapp@2.0.4'
+
+app({
+    view: () => h('body', {}, 'hello world'),
```

Comment:
@zaceno If this is about minimalism, then I'd just `view: () => "hello world"` and not even import `h`.

Reply:
Good point. Except then I’d have to also add a `<div id=“app”></div>` to the body and use `document.getElementById(‘app’)` for the node. I suppose that would be the preferred way to make a real app, but this is, in total, more minimal I think.

Reply:
Ah, missed `document.body`, in that case, could this be more minimal? 😄 

```js
import { app } from "https://unpkg.com/hyperapp@2.0.4"

app({ node, view: () => "hello world" })
```

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <title>Hello World</title>
    <script src="./hello-world.js" type="module"></script>
  </head>
  <body><p id="node"></p></body>
</html>
```

Reply:
Or we could just go all the way...

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <title>Hello World</title>
    <script type="module">
      import { app } from "https://unpkg.com/hyperapp@2.0.4"

      app({ node, view: () => "hello world" })
    </script>
  </head>
  <body>
    <x id="node"></x>
  </body>
</html>
```

Reply:
That doesn’t work though. Does it?! How is node globally available?

Reply:
[Ah, but it does!](https://html.spec.whatwg.org/multipage/window-object.html#named-access-on-the-window-object) :trollface:

Reply:
🤯 all these years on the web and I didn’t know!

Reply:
Cool. Ok Yeah it makes sense for hello world to just render hello world without rendering the body tags. Updated for that. Didn't use the fancy node-hack, cool as it is ;)